pallets/stake/Cargo.toml

```diff
@@ -6,9 +6,9 @@ edition = "2018"
 description = "staking pallet for validator selection and rewards"
 
 [dependencies]
+author = { path = "../author", default-features = false }
 frame-support = { git = "https://github.com/paritytech/substrate", branch = "master", default-features = false }
 frame-system = { git = "https://github.com/paritytech/substrate", branch = "master", default-features = false }
-pallet-authorship = { git = "https://github.com/paritytech/substrate", branch = "master", default-features = false }
```

Comment:
One feature that would be really helpful for playing with alive moonbeam network is if the Apps UI could display the block author. It does this for many chains already. It does not do this for Moonbeam.

I suspect that Apps pulls the authorship information from pallet authorship if it is present in the runtime. (@jacogr is that right?) If it is, it would be really helpful to make pallet authorship work with the authorship inherent.

Reply:
Author is extracted from the digest via the `PreRuntime` or `Consensus` items (depending on version of the chain), basically it happens here - https://github.com/polkadot-js/api/blob/master/packages/api-derive/src/type/util.ts

... which in turn links here for Aura/Babe (and actually Kulupu pow) - https://github.com/polkadot-js/api/blob/master/packages/types/src/generic/ConsensusEngineId.ts#L94

Reply:
@jacogr We are actually still using Consensus because we don't use Aura or Babe, so we would appreciate greatly if you continue supporting Consensus logs.

These are the logs we get on a block: 
<img width="623" alt="Screenshot 2021-01-19 at 20 04 08" src="https://user-images.githubusercontent.com/16758383/105081251-37a1ae80-5a92-11eb-8299-6b29807f2ae1.png">

If you allow, I can modify the code here (https://github.com/polkadot-js/api/blob/master/packages/api-derive/src/type/util.ts) to detect the author for moonbeam.

Thanks in advance

Reply:
Sure. (Sorry, seem to have missed the notification earlier)

Reply:
@jacogr
Do you mind reminding me the script to use a local version of api in a local version of the app? I can't figure it out with npm link
Also, can you explain to me how those script work/where they are located so that I don't have to bother you with that again?
Thanks in advance

Reply:
`yarn polkadot-dev-copy-to <destination>`, where destination is eg `apps`

Reply:
thanks